code/crates/test/app/src/app.rs

```diff
@@ -232,8 +260,13 @@ pub async fn run(state: &mut State, channels: &mut Channels<TestContext>) -> eyr
                             .get_validator_set(state.current_height)
                             .expect("Validator set should be available");
 
+                        let updates = Updates {
+                            validator_set: Some(validator_set),
```

Comment:
I think we can pass `None` here as we restart the same height with same valset as before.
```suggestion
                            validator_set: None,
```